dmv/dmv.js

```diff
@@ -19,13 +19,18 @@ DMV.prototype.currentCustomerFor = function(agent) {
 }
 
```

Comment:
My name if Jeff

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"